arkouda/array_view.py

```diff
@@ -440,11 +441,25 @@ def save(
             "Please use ak.ArrayView.to_hdf",
             DeprecationWarning,
         )
```

Comment:
Yes we do. Since we are moving to the schema using `to_hdf` and `to_parquet`. The separation  of IO occurred later than the HDF5 updates. Thus, I did not want to remove these functions yet.